app/js/main.js

```diff
@@ -556,9 +565,24 @@ function videoStop() {
 function _videoPlay() {
     "use strict";
     // Display each frame
-    removeFrameReelSelection();
+    _removeFrameReelSelection();
     playback.setAttribute("src", capturedFramesRaw[curPlayFrame]);
     statusBarCurFrame.innerHTML = curPlayFrame + 1;
+
+    // Display playback position indicator after each frame
+    _addFrameReelSelection(curPlayFrame + 1);
+    
+    if (curPlayFrame === 0) {
+        // Scroll to start when playback begins
+        frameReelArea.scrollLeft = 0;
+    } else if (curPlayFrame + 1 !== curFrame) {
```

Comment:
@le717 curFrame is not zero based because each time a frame is captured it is incremented this means if 20 frames have been captured curFrame = 20.